reinforcement_learning/include/sender.h

```diff
@@ -9,14 +10,20 @@ namespace reinforcement_learning {
   public:
     virtual int init(api_status* status) = 0;
 
-    int send(std::string&& data, api_status* status = nullptr)
+    int send_byte(std::vector<unsigned char>& data, api_status* status = nullptr)
```

Comment:
why we cannot have move semantics here? if moving is impossible, it probably should be at least const &.

Reply:
I tried but moving using && by vector is not supported. Let me know if there is a better way. I can change it to const &.

---
In reply to: [228194531](https://github.com/VowpalWabbit/vowpal_wabbit/pull/1653#discussion_r228194531) [](ancestors = 228194531)